metricshub-hardware/src/main/java/org/sentrysoftware/metricshub/hardware/MetricNormalizationService.java

```diff
@@ -109,7 +110,9 @@ public void run() {
 							.normalize(monitor);
 						break;
 					case "network":

```

Comment:
Instead of using hardcoded monitor names `network`, `physical_disk`, ...etc. Call `KnownMonitorType.<TYPE>.getKey()`